### PR TITLE
FIX: Widgets were not reloading (macOS)

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/gnghb9428njt4f511zfhbnz8kr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7542)

File: tests/setup.js [hunk 1]
```diff
@@ -146,6 +146,19 @@ jest.mock('react-native-share', () => {
   };
 });
 
+jest.mock('react-native-widget-center', () => {
```
Comment (Overtorment) [inline tests/setup.js:149]: isnt it enough to mock just `WidgetCommunication` ?

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/c74x1udb2yuf96emv2m06c2128?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7557)

File: tests/setup.js (hunk 1 above)
Comment (marcosrdz) [reply tests/setup.js]: Yeah. that was left over code when trying to fix the tests. Removed.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/dyqpc2y1mjabg1wb38z60zy308?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7558)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**